client/src/Journal.css

```diff
@@ -0,0 +1,60 @@
+.container {
+  max-width: 575px;
+  margin: 0 auto;
+  padding: 0 1.5rem;
+}
+
+.header {
+  background-color: #562b81;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-full,
+.column-half {
```

Comment:
remove `.column-half`, because this is higher in the cascade it's overridden by `lines 57-59` and thus redundant